### PR TITLE
Fixed things and cleaned up paths

### PR DESCRIPTION
**Pull request by ifohancroft**
I have fixed and/or cleaned up the following things:

-- Everything has been ungrouped and regrouped again so in theory there should no longer be a difference between the grouped and ungrouped versions.
-- Merged duplicated/overlapping nodes where I have found them.
-- Redone the clips where the objects from which they were made have been changed
-- Fixed a sharp point in the right foot
-- Fixed a bump and a sharp point in the left and right arms/body meeting points and a bump in the neck that were caused by path effect spiro spline and arms
-- Fixed right foot highlight that was sticking outside of the foot
-- Every path and group has been named appropriately (even the clip objects)
-- A couple more things that I forgot to mention.

TL;DR: This PR fixes all the bugs that were left and cleans up everything that was left to be cleaned up. I may have missed some duplicated/overlapping nodes but they weren't causing problems so I haven't noticed them and I have already fixed everything else so I didn't want to go through everything again to check.

I have re-created the other vector images and png exports from the main one and have added myself to the README.

### CODE REVIEW

**Comment by ifohancroft** [comment]
P.S. I have spoken with Sticker Mule about printing the fixed and cleaned up Tux from the PR in Full Bleed.
It is still not printable, mainly because of the shadows on Tux's feet. To be able to print designs in full bleed, their outer edges need to have 1/8th inch of flat color. However, I am not good with shadows and gradients and making things look good. Would you mind trying to make the parts of the feet marked in red be flat color about 1/8th of an inch deep? https://i.imgur.com/W3UR5EP.png
If you can make it look good and there aren't problems with the edges themselves being too complex (I am still trying to find that out, I sent them a version of Tux with flat black feet just to check if the feet shadows are the only problem) we can just have the change on Tux itself, no need for a separate sticker version.

**Comment by ifohancroft** [comment]
Bump :D

**Comment by garrett** [comment]
I just checked out the PR. Thanks! (I don't often check GitHub on the weekends and didn't get around to it yesterday; sorry for the delay.)

However, when ungrouping Tux in the main `tux.svg` file, Inkscape 1.0 crashes. :scream: Do you see the same behavior in Inkscape 1.0?

The other files work as expected.

**Comment by garrett** [comment]
Oh. I reset to what's in the master branch and I'm seeing the same behavior in Inkscape 1.0 (crashing).

**Comment by garrett** [comment]
Apparently, I was still on master when testing this. What's in the PR works... and master is apparently crashing Inkscape 1.0 for some reason. Whoops!

Meanwhile, in commit b66ea70aa6b23ad6d9837a68e9a5ea671dfb2045, the tux-bw-inverted.svg and png were both updated to look different from before. (The intent is to have a Tux on an dark background, and have an outline to protect how Tux looks.)

**Comment by ifohancroft** [comment]
Most welcome and no problem for the delay!
I haven't tested in 1.0 yet. I have just updated to 1.0 yesterday.
I am surprised the PR works in it, I thought the update will break pretty much all SVGs created in previous versions :D I am glad that's not the case.

As for tux-bw-inverted.svg and png, sorry, I didn't really understand that. Are you saying that they are ok like they are in the PR or that they are not? If it's the latter, what changes should I do to them?

Also, did you had a chance to check my comment above about the feet gradients?

**Comment by garrett** [comment]
Using the diff viewer @ https://github.com/garrett/Tux/pull/8/files#diff-bc5df85fa4a378ee83cbabb5928cfef7 show the difference — this affects both the SVG and PNG, of course.

Here's a screenshot:

Screenshot_2020-05-05 Fixed things and cleaned up paths by ifohancroft · Pull Request #8 · garrett Tux

Differences:
1. Tux's body changed from black to white
2. the outline was dropped
3. image size changed (not sure if this is intended or not... originally I had a bit of padding around tux, but I guess it's not really needed and might get in the way when used elsewhere)

The problems are 1 & 2.

**Comment by garrett** [comment]
As far as the gradient on the feet are concerned: That would change the way tux looks. Perhaps we should have a separate file for printing stickers? Like `tux-sticker.svg`? 

Ideally, yes, it would be nice if there could be just one file; I just want the default tux to look like the official one that Larry made. But if that's not good enough for printing stickers, we should have one that's authentic as possible and adjusted to be more optimal for printing.

**Comment by ifohancroft** [comment]
Ah yes! I missed that. I believe I was comparing to a png that may have been different. I will fix this in the next couple of hours the latest.
Is this just about the tux-bw-inverted svg and png?

As for point 4. I totally missed that. I usually prefer saving SVGs without padding but if you want, I can add some again?

Ah! Good point about keeping it as close to the original as possible.
Yeah, perhaps we should have a separate version for stickers. I am thinking the same as this one from the PR, just with the brown part of the left foot being flat color maybe and not quite sure what can be done for the right foot. Oh and I just remembered, there was one more thing regarding the stickers - the shadow on the right hand gets a bit too closer to the edge so maybe we can have it end a bit sooner on the sticker version?

**Comment by ifohancroft** [comment]
I still need to regenerate the rest of the Tux's but I am uploading a pdf from Sticker Mule that shows which part of Tux need to be flat for full bleed (what's outside of the blue line) and still only the feet are the problem, towards the outside, where they touch the body it's ok, but I think their PDF provides a better visual.

[tux-full-bleed.pdf](https://github.com/garrett/Tux/files/4594487/tux-full-bleed.pdf)

**Comment by ifohancroft** [comment]
Bump :D

**Comment by ifohancroft** [comment]
Bump :D

**Comment by garrett** [comment]
> Not sure why I got confused and got the inverted Tux inverted :D

It's that way in master too. Either I goofed up at some point, or it was changed in the previous PR (and I goofed up by letting it through without checking it). Oops.

I've adopted all the commits to tux.svg, tux-large.png, and the README file. Additionally, I squashed it all to one commit.

As for the other files, I moved those to another branch and will send that up as a PR.

I've split the two out, because I want the changes you made to Tux to be merged ASAP and we can go over what to do about the black and white version in a separate PR.

(I intentionally simplified the paths in the black and white version for better printing — especially at small sizes — and easier integration in other projects where needed. I'm thinking we have two versions: One more true, like yours, and a simplified version. But we can discuss this and work on it in the other PR I'm about to make.)

**Comment by garrett** [comment]
Thanks!

Merged. Continued in #9 for the BW variants.

**Comment by ifohancroft** [comment]
> Thanks!
> 
> Merged. Continued in #9 for the BW variants.

Thank you too! Will write in the other PR now :D

**Comment by garrett** [comment]
I've opened an issue for the sticker @ issue #10.
